packages/blade/src/components/Typography/Text/Text.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/display-name */
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import type { TextProps } from './';
+import TextComponent from './';
+//The Text component is a wrapper component that will apply typography styles to the text inside.
+
+const TextStoryMeta: Meta<TextProps<{ variant: 'body' | 'caption' }>> = {
+  title: 'Components/Typography/Text',
+  component: TextComponent,
+  args: {
+    variant: 'body',
+    weight: 'bold',
```

Comment:
Why bold by default? Feels weird. Keep regular only?

Reply:
was playing around and forgot to remove it 🤦